### PR TITLE
Added ProgressBar | Issue #879

### PR DESCRIPTION
**Pull request by Shariq2003**
Closes: #879 

- **Title:** Added ProgressBar
- **Name:** Shariq
- **Idenitfy yourself:** 
GSSOC Ext

### Describe the add-ons or changes you've made 📃

I have added a progress bar.

https://github.com/user-attachments/assets/81b618a1-413b-4dc6-9af0-5e25aea88850

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [X] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [X] This PR does not contain plagiarized content.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly wherever it was hard to understand.
- [X] My changes generate no new warnings.

### CODE REVIEW

**Comment by Shariq2003** [comment]
@Avdhesh-Varshney sir, please review and merge this

File: webmasterlog/src/app/globals.css
Comment (Avdhesh-Varshney) [inline webmasterlog/src/app/globals.css:1]: Add this css using tailwind css.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Nice Work. Make the same color of progressbar as of scrollbar

File: webmasterlog/src/app/globals.css
Comment (Shariq2003) [reply webmasterlog/src/app/globals.css:1]: > Add this css using tailwind css.

okay changing

**Comment by Shariq2003** [comment]
@Avdhesh-Varshney sir, please review and merge

**Comment by Shariq2003** [comment]
> Nice Work. Make the same color of progressbar as of scrollbar

Yes, its same

**Comment by Shariq2003** [comment]
@Avdhesh-Varshney sir, please review and merge this

**Review by Avdhesh-Varshney — APPROVED** [review]

**Comment by NEGRAH-S** [comment]
@MastanSayyad ,  only one file changed and readymade component added Level 1 should be assigned

**Comment by Shariq2003** [comment]
> @MastanSayyad , only one file changed and readymade component added Level 1 should be assigned

@NEGRAH-S Sir, in Webdev everything is already implemented somewhere that doesn't mean it is like a cherry on the cake.

**Review by MastanSayyad — COMMENTED** [review]
It's fine

**Pull request merged**